queries/vue/highlights.scm

```diff
@@ -1,7 +1,5 @@
 ; inherits: html_tags
 
-(dynamic_directive_inner_value) @tag
```

Comment:
We don't really need to inject nor highlight this with anything. The current vue parser/compiler (official vue compiler too) will fail if it's anything other than a variable name. Maybe `@variable` highlight for it